crates/sui-framework/sources/governance/staking_pool.move

```diff
@@ -30,11 +30,8 @@ module sui::staking_pool {
     const EPENDING_DELEGATION_DOES_NOT_EXIST: u64 = 8;
 
     /// A staking pool embedded in each validator struct in the system state object.
-    struct StakingPool has store {
-        /// The sui address of the validator associated with this pool.
-        validator_address: address,
-        /// The epoch at which this pool started operating. Should be the epoch at which the validator became active.
-        starting_epoch: u64,
+    struct StakingPool has key, store {
```

Comment:
Is there a reason for the `starting_epoch` to also be removed here? It  seems like something we'd want to keep in but I'm probably misunderstanding something :) Was the tuple of `(address, u64)` being used as some kind of UID for the staking pool before this?

Reply:
It does look like the `starting_epoch` needs to be kept because wallet / explorer depends on it to calculate APY. I'm adding it back.
> Was the tuple of (address, u64) being used as some kind of UID for the staking pool before this?

yup